lnbits/static/i18n/kr.js

```diff
@@ -0,0 +1,195 @@
+window.localisation.en = {
+  confirm: '확인',
+  server: '서버',
+  theme: '테마',
+  funding: '자금',
+  users: '사용자',
+  apps: '앱',
+  channels: '채널',
+  transactions: '거래 내역',
+  dashboard: '현황판',
+  manage_node: '노드 관리',
+  total_capacity: '총 용량',
+  avg_channel_size: '평균 채널 용량',
+  biggest_channel_size: '가장 큰 채널 용량',
+  smallest_channel_size: '가장 작은 채널 용량',
+  number_of_channels: '채널 수',
+  active_channels: '활성화된 채널',
+  connect_peer: '피어 연결하기',
+  connect: '연결하기',
+  open_channel: '채널 개설하기',
+  open: '개설',
+  close_channel: '채널 폐쇄하기',
+  close: '폐쇄',
+  restart: '서버 재시작',
+  save: '저장',
+  save_tooltip: '변경 사항 저장',
+  topup: '자금 추가',
+  topup_wallet: '지갑에 자금 추가',
+  topup_hint: '자금을 추가할 지갑의 ID를 넣어주세요',
+  restart_tooltip: '변경 사항을 적용하려면 서버를 재시작해야 합니다.',
+  add_funds_tooltip: '지갑에 자금을 추가합니다.',
+  reset_defaults: '기본 설정으로 돌아가기',
+  reset_defaults_tooltip: '설정했던 내용들을 모두 지우고, 기본 설정으로 돌아갑니다.',
+  download_backup: '데이터베이스 백업 다운로드',
+  name_your_wallet: '사용할 %{name}지갑의 이름을 정하세요',
+  paste_invoice_label: '인보이스, 결제 요청, 혹은 lnurl 코드를 붙여넣으세요 *',
+  lnbits_description:
+    '설정이 쉽고 가벼운 LNBits는 어떤 라이트닝 네트워크의 예산 자원 위에서든 돌아갈 수 있습니다. 현재 지원하는 예산 자원의 형태는 LND, Core Lightning, OpenNode, Alby, LNPay, 그리고 다른 LNBits 지갑들입니다. 스스로 사용하기 위해, 또는 다른 사람들에게 수탁형 솔루션을 제공하기 위해 LNBits를 운영할 수 있습니다. 각 지갑들은 자신만의 API key를 가지며, 생성 가능한 지갑의 수에는 제한이 없습니다. 자금을 분할할 수 있는 기능으로 인해, LNBits는 자금 운영 도구로써뿐만 아니라 개발 도구로써도 유용합니다. 확장 기능들은 LNBits에 여러분들이 라이트닝 네트워크의 다양한 최신 기술들을 수행해볼 수 있게 하는 추가 기능을 제공합니다. LNBits 개발진들은 확장 기능들의 개발 또한 가능한 쉽게 만들었으며, 무료 오픈 소스 프로젝트답게 사람들이 자신만의 확장 기능들을 개발하고 제출하기를 응원합니다.',
+  export_to_phone: 'QR 코드를 이용해 모바일 기기로 내보내기',
+  export_to_phone_desc:
+    '이 QR 코드는 선택된 지갑의 최대 접근 권한을 가진 전체 URL을 담고 있습니다. 스캔 후, 모바일 기기에서 지갑을 열 수 있습니다.'
+  wallets: '지갑',
+  add_wallet: '새로운 지갑을 추가합니다',
+  delete_wallet: '지갑을 삭제합니다',
+  delete_wallet_desc:
+    '이 지갑은 삭제될 것이며, 삭제 시 지갑 내 자금은 복구가 불가능합니다.',
+  rename_wallet: '지갑 이름 변경',
+  update_name: '이름 변경하기',
+  fiat_tracking: '법정통화 가격 표시',
+  currency: '통화',
+  update_currency: '통화 수정하기',
+  press_to_claim: '비트코인을 수령하려면 눌러주세요',
+  donate: '기부',
+  view_github: 'GitHub 페이지 보기',
+  voidwallet_active: 'VoidWallet이 활성화되었습니다! 결제가 불가능합니다.',
+  use_with_caution: '주의하세요 - %{name} 지갑은 아직 BETA 단계입니다.',
+  service_fee: '서비스 수수료: 거래액의 %{amount} %',
+  service_fee_max: '서비스 수수료: 거래액의 %{amount} % (최대 %{max} sats)',
+  service_fee_tooltip:
+    'Service fee charged by the LNbits server admin per outgoing transaction 지불 결제 시마다 LNBits 서버 관리자에게 납부되는 서비스 수수료',
```

Comment:
Why is the first part of the string untranslated?

Reply:
I missed the first line. Thanks!